WalletWasabi/Models/UpdateStatus.cs

```diff
@@ -2,21 +2,19 @@ namespace WalletWasabi.Models;
 
 public class UpdateStatus : IEquatable<UpdateStatus>
 {
-	public UpdateStatus(bool backendCompatible, bool clientUpToDate, ushort currentBackendMajorVersion, Version clientVersion)
+	public UpdateStatus(bool backendCompatible, ushort currentBackendMajorVersion)
 	{
 		BackendCompatible = backendCompatible;
-		ClientUpToDate = clientUpToDate;
 		CurrentBackendMajorVersion = currentBackendMajorVersion;
-		ClientVersion = clientVersion;
 	}
 
-	public bool ClientUpToDate { get; }
+	public bool ClientUpToDate { get; set; }
```

Comment:
We could simplify it yes, but to note:
When we instantiate UpdateStatus [here](https://github.com/zkSNACKs/WalletWasabi/pull/13027/files#diff-6c2ef708b9d17b5f1441eee88abf2ad9e743cd770a02b6e08047767fdfc0762fL234), we aren't aware of ClientVersion (API request needed), ReadyToInstall (download needed) and ClientUpToDate (client version comparison needed).
I went with this because this way we can set these properties in UpdateManager [when its needed](https://github.com/zkSNACKs/WalletWasabi/pull/13027/files#diff-3fc9c79accfd2dd9d82aca94e9b518b53671baa674a5f8206415070d56de6e15R69-R71).

Reply:
@kiminuo is this ok? 6edb57b